index.js

```diff
@@ -1,20 +1,20 @@
 'use strict';
 const net = require('net');
 
-const getPort = port => new Promise((resolve, reject) => {
+const getPort = (port, host) => new Promise((resolve, reject) => {
 	const server = net.createServer();
 
 	server.unref();
 	server.on('error', reject);
 
-	server.listen(port, () => {
+	server.listen({port, host}, () => {
```

Comment:
This is not correct. `server.listen` accepts port and host as arguments, not an object.